docs/po/pt.po

```diff
@@ -1590,6 +1590,1003 @@ msgstr "`/blockhash/<ALTURA>`: hash do bloco em uma determinada altura de bloco.
+msgstr ""
+"Uma output de transação pode conter saldos de qualquer número de runes. As runes são identificadas por IDs, que consistem no bloco em que uma rune foi gravada e no índice da transação de gravação dentro desse bloco, representados em texto como `BLOCO:TX`. "
+"Por exemplo, o ID da rune gravada na 20ª transação do 500º bloco é `500:20`."
+
+#: src/runes.md:33
+msgid "Etching"
+msgstr "Gravações"
+
+#: src/runes.md:36
+msgid "Runes come into existence by being etched. Etching creates a rune and sets its properties. Once set, these properties are immutable, even to its etcher."
+msgstr ""
+"As runes passam a existir ao serem gravadas. A gravação cria uma rune e estabelece as suas propriedades. Uma vez estabelecidas, estas propriedades são imutáveis, mesmo para o seu gravador."
+
+#: src/runes.md:39 src/runes/specification.md:420
+msgid "Name"
+msgstr "Nome"
+
+#: src/runes.md:41
+msgid "Names consist of the letters A through Z and are between one and twenty-six letters long. For example `UNCOMMONGOODS` is a rune name."
+msgstr "Os nomes consistem nas letras de A a Z e têm entre uma e vinte e seis letras de comprimento. Por exemplo, `UNCOMMONGOODS` é um nome de uma rune."
+
+#: src/runes.md:44
+msgid "Names may contain spacers, represented as bullets, to aid readability. `UNCOMMONGOODS` might be etched as `UNCOMMON•GOODS`."
+msgstr "Os nomes podem conter separadores, representados como pontos, para ajudar na legibilidade. `UNCOMMONGOODS` pode ser gravado como `UNCOMMON•GOODS`."
+
+#: src/runes.md:47
+msgid "The uniqueness of a name does not depend on spacers. Thus, a rune may not be etched with the same sequence of letters as an existing rune, even if it has different spacers."
+msgstr ""
+"A singularidade de um nome não depende dos separadores. Assim, uma rune não pode ser gravada com a mesma sequência de letras de uma rune existente, mesmo que "
+"tenha separadores diferentes."
+
+#: src/runes.md:51
+msgid "Spacers can only be placed between two letters. Finally, spacers do not count towards the letter count."
+msgstr "Os separadores só podem ser colocados entre duas letras. Por fim, os separadores não contam para o número total de letras."
+
+#: src/runes.md:54 src/runes/specification.md:330 src/runes/specification.md:338
+msgid "Divisibility"
+msgstr "Divisibilidade"
+
+#: src/runes.md:56
+msgid ""
+"A rune's divisibility is how finely it may be divided into its atomic units. Divisibility is expressed as the number of digits permissible after the decimal point in an amount of "
+"runes. A rune with divisibility 0 may not be divided. A unit of a rune with divisibility 1 may be divided into ten sub-units, a rune with divisibility 2 may be divided into a "
+"hundred, and so on."
+msgstr ""
+"A divisibilidade de uma rune é a finura com que pode ser dividida nas suas unidades atómicas. A divisibilidade é expressa como o número de dígitos permitidos após o ponto decimal "
```

Comment:
Change for:
A divisibilidade de uma rune **é o quanto que pode** ser dividida nas suas unidades atómicas.